src/components/widgets/DateTimeWidget.js

```diff
@@ -1,8 +1,27 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { pad } from "../../utils";
 
-function fromJSONDate(jsonDate) {
-  return jsonDate ? jsonDate.slice(0, 19) : "";
+export function fromJSONDate(jsonDate) {
```

Comment:
I know this is a bit outside the scope of the PR, but could you rename these functions something like `utcToLocal` and `localToUTC`?

Reply:
Happy to :)